docs/02-guides/customize-the-generated-sitemap.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: Customize the generated sitemap
+description:
+  In this guide you will learn how to customize the generated sitemap.
+---
+
+<SinceVersion tag="3.4" />
+
+<p>{frontMatter.description}</p>
+
+## Static Pages Customisation
+
+You can customize the sitemap for static pages by using the `getSitemapEntries`
+function in the SEO handle. This function allows you to add custom entries to
+the sitemap. You can use this function to add custom URLs to the sitemap, set
+the last modification date, change frequency, and priority for each URL.
+
+#### Example
+
+```tsx title="src/routes/blog.tsx"
+import { createSeoHandle } from "@front-commerce/remix/seo";
+
+export const handle = createSeoHandle({
+  getSitemapEntries: ({
+    request, // The Request instance for the `/sitemap.xml` endpoint.
+    app, // The FrontCommerceApp instance.
+  }) => {
```

Comment:
**nitpick:** instead of these comments, I think adding explicit types would be better

**suggestion:** add types to the params

Reply:
I will remove the comments, but I don't understand `add types to the params` there is 1 param which is an object containing request and app, do you mean with a link to the types?

Reply:
They are already typed in the `createSeoHandle`